.percy.yml

```diff
@@ -0,0 +1,4 @@
+version: 1
+snapshot:
+  widths: [1024]
```

Comment:
this isn't working

the docs say
```Percy Agent is primarily configured by creating a .percy.yml file. This yaml file allows you to globally set configuration for each build. The .percy.yml configuration file must exist in the directory where you run the percy from.

```

Maybe something to do with yarn workspaces? 🤔

Reply:
Ah, so it doesnt look for it upwards?